pieforproviders/pieforproviders
# [PIE-1034] Weekly Hours Attended from the API

Adds weekly_hours_attended for Nebraska dashboards using live algorithms, behind a feature flag.  Weeks start on Sunday.

## 💅🏼 What issue does this fix?


Closes #1034 

## 🍂 Before You Submit


## 🛷 Deployment


## 🧵 Steps to set up locally
```
bundle exec db:migrate
```


## 🧳 Steps to test

- add an active child approval with `authorized_weekly_hours`
- add several attendances for that child approval
- flip your .env feature flag to `FF_LIVE_ALGORITHMS_WEEKLY_HOURS_ATTENDED=true`
- start the server
- you should see the sum of your attendance time in the `hours_attended` field on the dashboard
- make sure the addition math is correct, rounded to one decimal place for the first number, and that the child_approval's authorized weekly hours displays correctly
- stop the server
- flip the feature flag to `false`
- add an "hours attended" value to the same child's temporary nebraska dashboard case
- start the server
- you should see the number you just added in the `hours_attended` field on the dashboard

## 🕯 Caveats, concerns
